ui/raidboss/data/06-ew/trial/endsinger-ex.txt

```diff
@@ -0,0 +1,119 @@
+65.4 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:702[02]:/
+74.5 "Telos" sync / 1[56]:[^:]*:The Endsinger:702E:/
+84.6 "Hubris" sync / 1[56]:[^:]*:The Endsinger:702C:/
+96.8 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6FF6:/
+110.2 "Star Collision" sync / 1[56]:[^:]*:[^:]+ Star:6FF[AB]:/
+111.8 "Diairesis" sync / 1[56]:[^:]*:The Endsinger:6FFC:/
+114.1 "Eironeia" sync / 1[56]:[^:]*:The Endsinger:702F:/
+115.1 "Eironeia" sync / 1[56]:[^:]*:The Endsinger:7030:/
+122.3 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6FFD:/
+132.6 "Star Collision" sync / 1[56]:[^:]*:[^:]+ Star:700[24]:/
+139.1 "Star Collision" sync / 1[56]:[^:]*:[^:]+ Star:700[24]:/
+145.4 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:702[02]:/
+
+# 5Head 1
+154.6 "Twinsong's Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:7007:/
+165.7 "Diairesis" sync / 1[56]:[^:]*:The Endsinger:7006:/
+174.8 "Diairesis" sync / 1[56]:[^:]*:The Endsinger:7006:/
+183.9 "Diairesis" sync / 1[56]:[^:]*:The Endsinger:6FFC:/
+192.0 "Theological Fatalism" sync / 1[56]:[^:]*:The Endsinger:700B:/
+205.2 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:702[02]:/
+205.2 "Eironeia" sync / 1[56]:[^:]*:The Endsinger:7030:/
+214.4 "Hubris" sync / 1[56]:[^:]*:The Endsinger:702C:/
+
+# Stack/Flare/Donut/Spread
+224.6 "Despair Unforgotten" sync / 1[56]:[^:]*:The Endsinger:7012:/
+232.1 "Benevolence" sync / 1[56]:[^:]*:The Endsinger:7016:/
+237.8 "Despair Unforgotten" sync / 1[56]:[^:]*:The Endsinger:7012:/
+245.3 "Befoulment" sync / 1[56]:[^:]*:The Endsinger:7014:/
+250.0 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:702[02]:/
+258.1 "Despair Unforgotten" sync / 1[56]:[^:]*:The Endsinger:7012:/
+265.6 "Benevolence" sync / 1[56]:[^:]*:The Endsinger:7016:/
+272.3 "Theological Fatalism" sync / 1[56]:[^:]*:The Endsinger:700B:/
+282.5 "Benevolence" sync / 1[56]:[^:]*:The Endsinger:701A:/
+
+289.5 "Telomania (cast)" sync / 1[56]:[^:]*:The Endsinger:72C3:/
+291.4 "Telomania (small) x4" duration 5 #sync / 1[56]:[^:]*:The Endsinger:665F:/
+299.7 "Telomania (big)" sync / 1[56]:[^:]*:The Endsinger:72C5:/
+
+# 6 Head Dodges 1
+313.6 "Endsong's Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:72B1:/
+321.8 "Endsong" sync / 1[56]:[^:]*:The Endsinger:701B:/
+332.3 "Endsong" sync / 1[56]:[^:]*:The Endsinger:701C:/
+341.4 "Endsong" sync / 1[56]:[^:]*:The Endsinger:701C:/
+350.5 "Endsong" sync / 1[56]:[^:]*:The Endsinger:701C:/
+354.0 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:702[02]:/
+363.1 "Telos" sync / 1[56]:[^:]*:The Endsinger:702E:/
+373.2 "Hubris" sync / 1[56]:[^:]*:The Endsinger:702C:/
+387.4 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6FF6:/
+400.8 "Star Collision" sync / 1[56]:[^:]*:[^:]+ Star:6FF[AB]:/
+402.4 "Diairesis" sync / 1[56]:[^:]*:The Endsinger:6FFC:/
+404.7 "Eironeia" sync / 1[56]:[^:]*:The Endsinger:702F:/
+405.7 "Eironeia" sync / 1[56]:[^:]*:The Endsinger:7030:/
+410.8 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6FFD:/
+420.9 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6FFD:/
```

Comment:
Is the first Eironeia boss-targeted and the second is damage? Maybe drop the 7030 if so? (Although it looks like you dropped 702F below? Usually I sync on boss abilities since I think that's when the snapshot for debuffs happens.)

Reply:
It's one on each healer, but if a healer is dead it doesn't actually go off, and they're slightly delayed from each other. Maybe a single line matching both with the earlier cast time?

Reply:
I think single line matching two things at different times (a second apart) isn't great.

What about single line only matching the first one but saying "Eironeia x2"?

Reply:
It has to match both because if a healer is dead, one of the two randomly won't go out. I guess it could just not sync instead?

Reply:
Nevermind, checking logs again, matching on just 2F is fine. Was just a broken log file due to messing with the betas that caused that one issue with 2F missing. This should be fine now.